### PR TITLE
Pass onLoad handler to Image and StandaloneImage

### PR DESCRIPTION
**Pull request by rodan-lewarx**

### CODE REVIEW

File: src/package/components/ChatBot/Plugin/Element/Image/index.jsx [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+  onLoad?: Function
 }
 
-export default function Image ({ src, title }) {
+export default function Image ({ src, title, onLoad }) {
+  const [isLoading, setIsLoading] = React.useState(true)
+  const handleOnLoad = () => setIsLoading(false)
+  React.useEffect(
+    () => {
+      console.info('loading', isLoading)
```
Comment (jindrichsamec) [inline src/package/components/ChatBot/Plugin/Element/Image/index.jsx]: Co tenhle console.info? :)

File: src/package/components/ChatBot/Plugin/Element/StandaloneImage/index.jsx [hunk 2]
```diff
@@ -4,20 +4,35 @@ import styles from './index.css'
...
+  onLoad?: Function,
 }
 
-export default function StandaloneImage ({ src, title }: Props): React.Node {
+export default function StandaloneImage ({ src, title, onLoad }: Props): React.Node {
+  const [isLoading, setIsLoading] = React.useState(true)
+  const handleOnLoad = () => setIsLoading(false)
+  React.useEffect(
+    () => {
+      console.info('loading', isLoading)
```
Comment (jindrichsamec) [inline src/package/components/ChatBot/Plugin/Element/StandaloneImage/index.jsx]: a tenhle?

File: src/package/components/ChatBot/Plugin/Element/Image/index.jsx (hunk 1 above)
Comment (rodan-lewarx) [reply src/package/components/ChatBot/Plugin/Element/Image/index.jsx]: @jindrichsamec fixed 🤦‍♂️

File: src/package/components/ChatBot/Plugin/Element/StandaloneImage/index.jsx (hunk 2 above)
Comment (rodan-lewarx) [reply src/package/components/ChatBot/Plugin/Element/StandaloneImage/index.jsx]: @jindrichsamec fixed 🤦‍♂️

**Pull request merged**